### PR TITLE
Convert tests to unittest

### PR DESCRIPTION
**Pull request by MichaelJFishman**
Split out from #67 .

### CODE REVIEW

File: pddlgym/tests/test_searchandrescue.py [hunk 1]
```diff
@@ -1,91 +1,93 @@
...
-    rng = np.random.RandomState(0)
-    for level in [1, 2]:
-        env = pddlgym.make(f"SearchAndRescueLevel{level}-v0")
-        for idx in range(len(env.problems)):
-            env.fix_problem_index(idx)
+import unittest
+
+
+class TestSearchAndRescue(unittest.TestCase):
+    def skip_test_searchandrescue(self, num_actions_to_test=10, verbose=False):
```
Comment (tomsilver) [inline pddlgym/tests/test_searchandrescue.py:8]: why is this skipped? oh, I guess we were skipping it already. fair enough!

File: pddlgym/tests/test_system.py [hunk 2]
```diff
@@ -3,11 +3,15 @@
...
+class TestSystem(unittest.TestCase):
+    def test_system(self):
+        print("WARNING: this test may take around a minute...")
+        run_all(render=False, verbose=False)
+        print("Test passed.")
 
 
 if __name__ == '__main__':
-    test_system()
+    unittest.main()
```
Comment (tomsilver) [inline pddlgym/tests/test_system.py:17]: add new line

**Review by tomsilver — APPROVED** [review]
Thanks so much @MichaelJFishman !

File: pddlgym/tests/test_searchandrescue.py (hunk 1 above)
Comment (MichaelJFishman) [reply pddlgym/tests/test_searchandrescue.py:8]: Yup! The calling line was previously commented out. I figured you wanted the code present, but unused, and making this NOT start with `test_` makes unittest skip it by default.

**Pull request merged**